.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Good job putting the `test.md` file inside your `.gitignore` file, but remember to put `node_modules` in your `.gitignore` file as well, as they can be included in your project while setting up your linters by accident, and you don't want those files in your GitHub.